gateway_test.go

```diff
@@ -241,7 +248,7 @@ var testsFallthrough = []FallthroughCase{
 }
 
 var testServiceIndexes = map[string][]net.IP{
-	"svc1.ns1": {net.ParseIP("192.0.1.1")},
+	"svc1.ns1": {net.ParseIP("192.0.1.1"), net.ParseIP("fd12:3456:789a:1::")},
```

Comment:
is it possible to add a test case with an invalid v6 address to make sure it doesn't end up in the response?

Reply:
Good catch, will look into making it more safe.

Reply:
Unfortunately this is not possible as `netip.ParseAddr` does not allow invalid ipv6 addresses. Therefore it should be sorted out during parsing of the address from the resources itself.

Reply:
yep, makes sense. and we don't really check IPs in the `kubernetes_test.go` anyway.